README.md

```diff
@@ -1,2 +1,37 @@
-# COMP4350-Project
+# ![logo](assets/Logo2.png)BisonCoin
+
 Repo containing documents and code for our COMP 4350 Group Project.
+
+## Members
+
+- Abhi Sachdev ([GitHub](https://github.com/abhisachdev17))
+- Madison Fines ([GitHub](https://github.com/madisonfines))
+- Akshay Sharma ([GitHub](https://github.com/akshaysharma21))
+- Zack Holmberg ([GitHub](https://github.com/ZackHolmberg))
+
+## Project Description
+
+- BisonCoin is a new cryptocurrency intended for use by University of Manitoba students to exchange on campus between each other or a variety of products at all campus businesses, including all food outlets and the book store!
+
+- The system will be composed of:
+  - A front-end web application
+  - A collection of microservices
+  - A Blockchain, cryptocurrency implementation
+  - Data storage, if needed
+
+## Core Features
+
+- With the BisonCoin app, students will be able to send and receive to their peers, as well as businesses in order to purchase products.
```

Comment:
Make it clear that they are sending and receiving "BisonCoin"

Reply:
Also, I don't think we need the first part saying "with the BisonCoin app"

Reply:
Great points, updated!